sources/packages/backend/libs/sims-db/src/entities/application.model.ts

```diff
@@ -333,6 +334,19 @@ export class Application extends RecordDataModel {
     nullable: true,
   })
   submittedDate?: Date;
+  /**
+   * Scholastic standings changes associated with this application.
+   */
+  @OneToMany(
+    () => StudentScholasticStanding,
+    (studentScholasticStanding) => studentScholasticStanding.application,
+    {
+      eager: false,
+      cascade: false,
```

Comment:
Not about this PR. cascade by default is false. should we mention explicitly? I know we are doing this in many places. Me be we can stop defining explicitly.

Reply:
You are right, the default is false, but we have been defining the relations like this for quite sometime.

Reply:
Yes, I am aware, I just kept doing what we have been doing 😉